diff --git a/lib/dune b/lib/dune
index 8c49f02..fd83e4c 100644
--- a/lib/dune
+++ b/lib/dune
@@ -15,4 +15,4 @@
   uri
   str
   xmlm)
- (wrapped false))
+ (wrapped true))
diff --git a/lib/mldoc.ml b/lib/mldoc.ml
index d50ef61..51eeebd 100644
--- a/lib/mldoc.ml
+++ b/lib/mldoc.ml
@@ -3,6 +3,8 @@
 module Document = Document
 module Block = Type_parser.Block
 module Inline = Inline
+module Nested_link = Nested_link
+module Pos = Pos
 module Exporters = Exporter.Exporters
 module Conf = Conf
 module Exporter = Exporter
